packages/components/src/components/button-group/readme.md

```diff
@@ -6,6 +6,10 @@ Die **ButtonGroup**-Komponente fasst mehrere Buttons zu einer optischen und funk
 
 ## Konstruktion
 
+### Hinweis zur Version 3
+
+Die **ButtonGroup** Komponente der Version 3 wurde als veraltet markiert (`deprecated`), da sie keinen nennenswerten Beitrag zur Verbesserung der Barrierefreiheit beiträgt.
```

Comment:
Die Komponente ist nicht in der Version 3, sondern für Version 3 abgekündigt.

Aktuell sind wir in Version 2.